File: app/src/main/java/com/apps/adrcotfas/goodtime/statistics/Main/PieChartGestureListener.java
package com.apps.adrcotfas.goodtime.statistics.Main;

import android.view.MotionEvent;

import com.github.mikephil.charting.listener.ChartTouchListener;
import com.github.mikephil.charting.listener.OnChartGestureListener;

public class PieChartGestureListener implements OnChartGestureListener {

    @Override
    public void onChartSingleTapped(MotionEvent me) {}
    @Override
    public void onChartGestureStart(MotionEvent me, ChartTouchListener.ChartGesture lastPerformedGesture) {}
    @Override
    public void onChartGestureEnd(MotionEvent me, ChartTouchListener.ChartGesture lastPerformedGesture) {}
    @Override
    public void onChartLongPressed(MotionEvent me) {}
    @Override
    public void onChartDoubleTapped(MotionEvent me) {}
    @Override
    public void onChartFling(MotionEvent me1, MotionEvent me2, float velocityX, float velocityY) {}
    @Override
    public void onChartScale(MotionEvent me, float scaleX, float scaleY) {}
    @Override
    public void onChartTranslate(MotionEvent me, float dX, float dY) {}
}

File: app/src/main/java/com/apps/adrcotfas/goodtime/statistics/main/PieChartGestureListener.kt
package com.apps.adrcotfas.goodtime.statistics.main

import android.view.MotionEvent
import com.github.mikephil.charting.listener.ChartTouchListener.ChartGesture
import com.github.mikephil.charting.listener.OnChartGestureListener

open class PieChartGestureListener : OnChartGestureListener {
    override fun onChartSingleTapped(me: MotionEvent) {}
    override fun onChartGestureStart(me: MotionEvent, lastPerformedGesture: ChartGesture) {}
    override fun onChartGestureEnd(me: MotionEvent, lastPerformedGesture: ChartGesture) {}
    override fun onChartLongPressed(me: MotionEvent) {}
    override fun onChartDoubleTapped(me: MotionEvent) {}
    override fun onChartFling(
        me1: MotionEvent,
        me2: MotionEvent,
        velocityX: Float,
        velocityY: Float
    ) {
    }

    override fun onChartScale(me: MotionEvent, scaleX: Float, scaleY: Float) {}
    override fun onChartTranslate(me: MotionEvent, dX: Float, dY: Float) {}
}